webview/src/plots/components/PlotsContainer.tsx

```diff
@@ -204,19 +222,36 @@ export const PlotsContainer: React.FC<PlotsContainerProps> = ({
             </div>
           </Tooltip>
         </summary>
-        {changeNbItemsPerRow && hasItems && maxNbPlotsPerRow > 1 && (
-          <div
-            className={styles.nbItemsPerRowSlider}
-            data-testid="nb-items-per-row-slider"
-          >
-            <MinMaxSlider
-              maximum={-1}
-              minimum={-maxNbPlotsPerRow}
-              label="Plot Width"
-              onChange={handleResize}
-              defaultValue={-nbItemsPerRow}
-            />
-          </div>
+        {changeSize && hasItems && maxNbPlotsPerRow > 1 && (
+          <>
+            <div
+              className={styles.sizeSliders}
+              data-testid="nb-items-per-row-slider"
+            >
+              <div className={styles.sizeSlider}>
```

Comment:
Maybe it's just me, but it seems the two sliders on top of each other cause the header to have some empty space 🤔 

What about having them side by side? 

<img width="1467" alt="Screenshot 2023-03-12 at 4 43 55 PM" src="https://user-images.githubusercontent.com/43496356/224575531-18598251-1d5a-437c-b267-3b581c7415bc.png">

Reply:
Could be complicated but I think we should consolidate everything into the section header and make it sticky (as per https://github.com/iterative/vscode-dvc/pull/3443#pullrequestreview-1336087886). WDYT?

Reply:
I think it'll make the whole thing too bulky. I also like the fact that when we collapse the section, we don't see the sliders anymore. It's more of a distraction otherwise. Having them side by side isn't a bad idea. But I'll make the headers sticky as well.